Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ButtonData.xml

```diff
@@ -7082,4 +7082,125 @@
         <EditorCategories value="Race:Protoss"/>
         <HotkeyAlias value="AP_ArchonSiphon"/>
     </CButton>
+    <CButton id="AP_ScoutNerazimCloak">
+        <Icon value="AP\Assets\Custom\Textures\btn-ability-terran-cloak-gold.dds"/>
+        <EditorCategories value="Race:Protoss"/>
+        <HotkeyAlias value="CloakOff"/>
+        <HotkeySet value="Cloak"/>
+    </CButton>
+    <CButton id="AP_ScoutNerazimCloakUpgrade">
+        <Icon value="AP\Assets\Custom\Textures\btn-ability-terran-cloak-gold.dds"/>
+        <EditorCategories value="Race:Protoss"/>
+        <HotkeyAlias value="CloakOff"/>
+        <HotkeySet value="Cloak"/>
+    </CButton>
+    <CButton id="AP_ScoutPurifierSolariteReactor">
+        <Icon value="Assets\Textures\btn-ability-zeratul-darkarchon-maelstrom.dds"/>
+        <AlertIcon value="Assets\Textures\btn-ability-zeratul-darkarchon-maelstrom.dds"/>
+        <EditorCategories value="Race:Protoss"/>
+        <HotkeyAlias value="AP_ScoutTaldarim"/>
+    </CButton>
+    <CButton id="AP_ScoutTaldarimAdvancedPhotonBlasters">
+        <Icon value="Assets\Textures\btn-upgrade-protoss-airweaponslevel3.dds"/>
+        <AlertIcon value="Assets\Textures\btn-upgrade-protoss-airweaponslevel3.dds"/>
+        <EditorCategories value="Race:Protoss"/>
+        <HotkeyAlias value="AP_ScoutAdvancedPhotonBlasters"/>
+    </CButton>
+    <CButton id="AP_ScoutNerazim">
+        <Icon value="AP\Assets\Textures\btn-unit-protoss-scoutnerazim.dds"/>
+        <AlertIcon value="AP\Assets\Textures\btn-unit-protoss-scoutnerazim.dds"/>
+        <EditorCategories value="Race:Protoss"/>
+        <HotkeyAlias value="AP_Scout"/>
+        <Hotkey value="Button/Hotkey/AP_Scout"/>
+    </CButton>
+    <CButton id="AP_ScoutNerazimPilot">
+        <Icon value="Assets\Textures\btn-ability-protoss-veilofshadowsvorazun.dds"/>
+        <EditorCategories value="Race:Protoss"/>
+    </CButton>
+    <CButton id="AP_ScoutNerazimPilotUnload">
+        <Icon value="Assets\Textures\btn-ability-terran-unloadall.dds"/>
+        <EditorCategories value="Race:Protoss"/>
+        <HotkeyAlias value="AP_ScoutNerazimPhantomDash"/>

```

Comment:
This button is normally race-tinted

What's doing the HotkeyAlias there? Could cause trouble with rebinding this